etc/cargo.config.prod

```diff
@@ -0,0 +1,14 @@
+# Configuration for release builds, that is bulids that are
+# _literally_ for packaging and releasing; and cases where we _must_
+# benchmark against a fully-optimized build.
+
+[profile.release]
+opt-level = 3
+debug = false
+rpath = false
+lto = true
+debug-assertions = false
+codegen-units = 1
+panic = 'unwind'
```

Comment:
TiKV has its own panic hook (unwind to log), maybe we can just set it to `panic=abort`?

Reply:
I think we might indeed be able to turn on panic=abort in production, though I'd like to look at it closer before doing so. Mostly I want to make sure that the way the abort happens looks intentional from a user perspective and not just like a SIGILL or something else alarming (I'm thinking of e.g. calling process::exit from the panic hook); but also I'd like to verify that there is no scenario where production tikv needs unwinding, including in tikv-ctl and tikv-importer, which are presumably less carefully tested than tikv-server.

We can definitely turn on panic=abort for any build that is just for benchmarking.